content/shell/BUILD.gn

```diff
@@ -439,6 +439,17 @@ static_library("content_shell_lib") {
   if (use_ozone) {
     deps += [ "//ui/ozone" ]
   }
+
+  if (is_cobalt) {
+    import("//starboard/build/buildflags.gni")
```

Comment:
Consider to put all import at the beginning of BUILD.gn.

Reply:
Done.